modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language_fr.properties

```diff
@@ -14,13 +14,13 @@ application-information=Informations relatives à l'application
 application-name=Nom
 application-name-help=Fournissez un nom pour identifier votre application pendant le processus d'autorisation.
 application-x-privacy-policy-x={0}Politique de confidentialité{1} de l'application
-are-you-sure-you-want-to-delete-the-application-there-are-no-authorizations-or-associated-tokens=Voulez-vous vraiment supprimer l'application ? Il n'y a aucune autorisation ou jeton associé.
-are-you-sure-you-want-to-delete-the-application-this-action-revokes-one-authorization-and-associated-tokens=Voulez-vous vraiment supprimer l'application ? Cette action révoque 1 autorisation et les jetons associés.
-are-you-sure-you-want-to-delete-the-application-this-action-revokes-x-authorizations-and-associated-tokens=Voulez-vous vraiment supprimer l'application ? Cette action révoque {0} autorisations et les jetons associés.
-are-you-sure-you-want-to-delete-the-selected-entries-this-action-revokes-all-authorizations-and-associated-tokens=Voulez-vous vraiment supprimer l'application ? Cette action révoque toutes les autorisations et les jetons associés.
+are-you-sure-you-want-to-delete-the-application-there-are-no-authorizations-or-associated-tokens=Voulez-vous vraiment supprimer l'application ? Il n'y a aucune autorisation ou jeton associé.
+are-you-sure-you-want-to-delete-the-application-this-action-revokes-one-authorization-and-associated-tokens=Voulez-vous vraiment supprimer l'application ? Cette action révoque 1 autorisation et les jetons associés.
+are-you-sure-you-want-to-delete-the-application-this-action-revokes-x-authorizations-and-associated-tokens=Voulez-vous vraiment supprimer l'application ? Cette action révoque {0} autorisations et les jetons associés.
+are-you-sure-you-want-to-delete-the-selected-entries-this-action-revokes-all-authorizations-and-associated-tokens=Voulez-vous vraiment supprimer l'application ? Cette action révoque toutes les autorisations et les jetons associés.
 are-you-sure-you-want-to-remove-access-for-the-selected-entries=Voulez-vous vraiment supprimer les accès aux entrées sélectionnées ?
 are-you-sure-you-want-to-revoke-the-authorization=Voulez-vous vraiment supprimer l'autorisation ?
-are-you-sure-you-want-to-revoke-the-selected-authorizations-they-will-be-revoked-immediately=Voulez-vous vraiment révoquer les autorisations sélectionnées ? Elles seront immédiatement abrogées.
+are-you-sure-you-want-to-revoke-the-selected-authorizations-they-will-be-revoked-immediately=Voulez-vous vraiment révoquer les autorisations sélectionnées ? Elles seront immédiatement abrogées.
```

Comment:
Unsure if `0xa0` (non-breaking space) is critical here, to avoid breaking lines.